engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/details/views/content-view-publish.html

```diff
@@ -7,6 +7,10 @@ <h3 translate>Publish New Version</h3>
   It can be promoted to other environments from the Versions tab of this Content View.
 </p>
 
+<div bst-alert="warning" ng-show="contentView.errors['messages'].length > 0">
+  <p translate>{{contentView.errors['messages'][0]}}</p>
+  </div>
```

Comment:
using a <p> causes the icon to be ona  new line:

![image](https://user-images.githubusercontent.com/395077/101052047-29381b80-3554-11eb-97bb-43e73230df97.png)

changing to a span seems to fix this